src/main/java/core/basesyntax/service/impl/ParserTransactionImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ParserService;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ParserTransactionImpl implements ParserService {
```

Comment:
The name should match the interface
```suggestion
public class ParserServiceImpl implements ParserService {
```